api/desecapi/tests/test_rrsets_bulk.py

```diff
@@ -166,7 +166,7 @@ def test_bulk_patch_fresh_rrsets_need_type(self):
 
     def test_bulk_patch_does_not_accept_single_objects(self):
         response = self.client.bulk_patch_rr_sets(domain_name=self.my_empty_domain.name, payload=self.data[0])
-        self.assertContains(response, 'expected a list of RRsets.', status_code=status.HTTP_400_BAD_REQUEST)
+        self.assertContains(response, 'Expected a list of items but got type ', status_code=status.HTTP_400_BAD_REQUEST)
```

Comment:
avoid the word `type`, as it could be confused with the resource record type

Reply:
fixed